helpers/mongodb

```diff
@@ -0,0 +1,360 @@
+#
+# usage: ynh_mongo_setup_db --db_user=user --db_name=name [--db_pwd=pwd]
+# | arg: -u, --db_user=		- Owner of the database
+# | arg: -n, --db_name=		- Name of the database
+# | arg: -p, --db_pwd=		- Password of the database. If not provided, a password will be generated
+#
+# After executing this helper, the password of the created database will be available in $db_pwd
+# It will also be stored as "mongopwd" into the app settings.
+#
+#
+ynh_mongo_setup_db() {
+	# Declare an array to define the options of this helper.
+	local legacy_args=unp
+	local -A args_array=( [u]=db_user= [n]=db_name= [p]=db_pwd= )
+	local db_user
+	local db_name
+	db_pwd=""
+	# Manage arguments with getopts
+	ynh_handle_getopts_args "$@"
+
+	local new_db_pwd=$(ynh_string_random) # Generate a random password
+	# If $db_pwd is not provided, use new_db_pwd instead for db_pwd
+	db_pwd="${db_pwd:-$new_db_pwd}"
+
+	# Create the user and grant access to the database
+	ynh_mongo_create_user --db_user="$db_user" --db_pwd="$db_pwd" --db_name="$db_name"
+
+	# Store the password in the app's config
+	ynh_app_setting_set --app=$app --key=db_pwd --value=$db_pwd
+}
+
+# Remove a database if it exists, and the associated user
+#
+# usage: ynh_mongo_remove_db --db_user=user --db_name=name
+# | arg: -u, --db_user=		- Owner of the database
+# | arg: -n, --db_name=		- Name of the database
+#
+#
+ynh_mongo_remove_db() {
+	# Declare an array to define the options of this helper.
+	local legacy_args=un
+	local -A args_array=( [u]=db_user= [n]=db_name= )
+	local db_user
+	local db_name
+	# Manage arguments with getopts
+	ynh_handle_getopts_args "$@"
+
+	if ynh_mongo_database_exists --database=$db_name; then  # Check if the database exists
+		ynh_mongo_drop_db --database=$db_name  # Remove the database
+	else
+		ynh_print_warn --message="Database $db_name not found"
+	fi
+
+	# Remove mongo user if it exists
+	ynh_mongo_drop_user --db_user=$db_user --db_name=$db_name
+}
+
+# Install MongoDB and integrate MongoDB service in YunoHost
+#
+# usage: ynh_install_mongo [--mongo_version=mongo_version]
+# | arg: -m, --mongo_version=		- Version of MongoDB to install
+#
+#
+ynh_install_mongo() {
+	# Declare an array to define the options of this helper.
+	local legacy_args=m
+	local -A args_array=([m]=mongo_version=)
+	local mongo_version
+	# Manage arguments with getopts
+	ynh_handle_getopts_args "$@"
+	mongo_version="${mongo_version:-$YNH_MONGO_VERSION}"
+
+	ynh_print_info --message="Installing MongoDB Community Edition ..."
+	local mongo_debian_release=$(ynh_get_debian_release)
+
+	if [[ $(cat /proc/cpuinfo) != *"avx"* && "$mongo_version" != "4.4" ]]; then
+    ynh_print_warn --message="Installing Mongo 4.4 as $mongo_version is not compatible with your cpu (see https://docs.mongodb.com/manual/administration/production-notes/#x86_64)."
+    mongo_version="4.4"
+  fi
+  if [[ "$mongo_version" == "4.4" && "$mongo_debian_release" != "buster" ]]; then
```

Comment:
```suggestion
  if [[ "$mongo_version" == "4.4" ]]; then
```

($mongo_debian_release is initialized with the debian version of the system, which is not going to be buster)